gen/gen.go

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"text/template"
+
+	"github.com/fatih/structs"
```

Comment:
We should not include unmaintained go modules.

Reply:
I couldn't find any other go module that does this. I we ok copying the code and NOT use the import?

Reply:
Good call! 🌟

Reply:
Updated to use https://github.com/go-viper/mapstructure